utils/cliutils/commandsflags.go

```diff
@@ -1782,7 +1782,7 @@ var commandFlags = map[string][]string{
 	},
 	BuildPublish: {
 		url, user, password, accessToken, sshPassphrase, sshKeyPath, serverId, buildUrl, bpDryRun,
-		envInclude, envExclude, InsecureTls, Project, bpDetailedSummary,
+		envInclude, envExclude, InsecureTls, Project, bpDetailedSummary, Overwrite,
```

Comment:
Since we are using the existing `Overwrite` flag, the current description will be shown in the help:
`[Default: false] Overwrites the instance configuration if an instance with the same ID already exists.`

I suggest introducing a new flag in the `flagsMap` with an accurate description of the functionality it adds. It can have a different index in the map, but the same flag name.